content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -280,7 +280,7 @@ define readiness distinct from completion. This is enforced during validation.
 
 Use `activeDeadlineSeconds` on the Pod and `livenessProbe` on the container to
 prevent init containers from failing forever. The active deadline includes init
-containers.
+containers. However it is recommended to use `activeDeadlineSeconds` if user deploy their application as a Job, because 'activeDeadlineSeconds' has an effect even after initContainer finished. The Pod which is already running correctly would be killed by 'activeDeadlineSeconds' if you set.
```

Comment:
I think it's better to change single quotes to back quotes.

```
Update from 'activeDeadlineSeconds' to `activeDeadlineSeconds`
```

Reply:
will update.

Reply:
updated.